dataValidator/helpers/dataLoader/asyncValidateDir.test.js

```diff
@@ -32,7 +32,7 @@ describe('asyncValidateDir helper', () => {
 
   it('should call readScenario for every file in the /data directory', async () => {
     const readScenarioSpy = jest.spyOn(readScenario, 'readScenario');
-    await expectMethodToBeCalledTimes(299, readScenarioSpy);
+    await expectMethodToBeCalledTimes(295, readScenarioSpy);
```

Comment:
confused how 4 assets have been deleted

Reply:
Should this PR still be in CR?

Reply:
Trying to find out where the 4 assets are.

Reply:
@pjlee11 I have looked at the diff and also confirmed with the `latest` branch. There is only one fixture data that was deleted. `data/afrique/featureIndex/48465371.json`. I deleted it because it was already in the `mediaAssetsPages`.